data/json/items/armor/jewelry.json

```diff
@@ -2784,6 +2914,14 @@
     "description": "A shiny, gold necklace adorned with a pearl pendant.",
     "color": "white"
   },
+  {
+    "id": "onyx_gold_pendant_necklace",
+    "copy-from": "garnet_gold_pendant_necklace",
+    "type": "ARMOR",
+    "name": { "str": "onyx and gold pendant necklace" },
+    "description": "A shiny, gold necklace adorned with an onyx pendant.",
```

Comment:
```suggestion
    "description": "A shiny gold necklace adorned with an onyx pendant.",
```

Reply:
I don't mind the wording without comma, but shouldn't other descriptions be adjusted too, then? It's currently the only necklace lacking comma:
![screenshot_2022-04-15-204805](https://user-images.githubusercontent.com/4951331/163612319-8a83156e-c7c4-4153-aa98-73ad69446ad9.png)